src/Mvc/Mvc.ApiExplorer/test/DefaultApiDescriptionProviderTest.cs

```diff
@@ -521,7 +540,7 @@ public static TheoryData ReturnsActionResultWithProducesAndProducesContentTypeDa
                     },
                     {
                         typeof(DefaultApiDescriptionProviderTest),
-                        nameof(DefaultApiDescriptionProviderTest.ReturnsActionResult),
+                        nameof(DefaultApiDescriptionProviderTest.ReturnsValueTaskOfActionResult),
```

Comment:
`DefaultApiDescriptionProviderTest.ReturnsActionResult` already gets tested a couple of lines down - it was a duplicate test. I removed the duplicate and added the new one for ValueTask.